runtime/lua/vim/lsp/handlers.lua

```diff
@@ -90,6 +90,7 @@ RSC[ms.window_showMessageRequest] = function(_, params)
   local co, is_main = coroutine.running()
   if co and not is_main then
     local opts = {
+      kind = 'lsp_message',
```

Comment:
Not sure what the existing precedent is, should we use dot instead? 

```suggestion
      kind = 'lsp.message',
```

Reply:
We don't have a precedent, other `kind`s simply don't use a word delimiter (e.g. `codeaction`). Maybe we can set this to `lspmessage`?

Reply:
Not sure. Not a blocker.

Reply:
In that case are we okay with merging this? :)